database/scripts/oqt-regions/shdi.sql

```diff
@@ -0,0 +1,46 @@
+/* Subnational HDI for regions within a country */
+WITH regions_shdi AS (
+    SELECT DISTINCT ON (ogc_fid)
+        shdi.shdi,
+        regions.ogc_fid AS ogc_fid
+    FROM
+        shdi,
+        regions
+    WHERE
+        st_intersects (shdi.wkb_geometry, regions.geom)
```

Comment:
what happens here if there are multiple shdi values per region. E.g. when the boundaries do not align properly? did you test this for a region "cross-country" region?

Reply:
Yes, thats the beauty of this SQL query. If multiple features are in the intersection the feature with the biggest intersection area will be selected. The trick here is the `select distinct on (ogc_fid)` and the `order by` clause.

I will add a comment in this script.

Reply:
And no I did not test this with regions which are truly cross country (meaning having a significat intersection area with both or more countries). This is not the case for the  current regions. But I will add a additional comment to make this clear in this file.

Reply:
I added comments to the code